Sources/SnapshotTesting/SnapshotsTestTrait.swift

```diff
@@ -0,0 +1,50 @@
+#if canImport(Testing)
+  @_spi(Experimental) import Testing
+
+  @_spi(Experimental)
+  extension Trait where Self == _SnapshotsTestTrait {
+    /// Configure snapshot testing in a suite or test.
+    /// - Parameters:
+    ///   - diffTool: The diff tool to use in failure messages.
+    ///   - record: The record mode of the test.
+    public static func snapshots(
+      diffTool: SnapshotTestingConfiguration.DiffTool = .default,
+      record: SnapshotTestingConfiguration.Record = .missing
```

Comment:
@mbrandonw Do you know if traits cascade and if this means these should be optional so that `.missing` won't get applied where a parent set to `.all`?